src/Mod/Arch/exportIFC.py

```diff
@@ -55,7 +55,7 @@
 __url__    = "https://www.freecad.org"
 
 # Save the Python open function because it will be redefined
-if open.__module__ in ['__builtin__', 'io']:
+if open.__module__ in ['__builtin__', 'io', '_io']:
```

Comment:
I would suggest using tuples instead of lists. Tuples are faster to create, faster to check for membership, and take up less space. (although these are all minor differences for a list of 3 elements, I'd still consider it good practice)
```suggestion
if open.__module__ in ('__builtin__', 'io', '_io'):
```